### PR TITLE
Dev/164 Delete student modal

### PR DESCRIPTION
**Pull request by Chaoyuuu**
Resolve: #164
image

### CODE REVIEW

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Review by Wally5077 — APPROVED** [review]
LGTM

File: src/services/StudentService.js [hunk 1]
```diff
@@ -116,6 +116,10 @@ export default class StudentService extends AbstractService {
             .then(res => new Group(res.data));
     }
 
+    async deleteStudentById(studentId) {
+        return this.axios.delete(`/api/students/${studentId}`)
```
Comment (Johnny850807) [inline src/services/StudentService.js:120]: semicolon

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Review done

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Pull request merged**